src/test/index.node.ts

```diff
@@ -187,11 +189,14 @@ export async function run(): Promise<void> {
     // Setup test files that need to be run.
     testFiles.forEach((file) => mocha.addFile(path.join(testsRoot, file)));
 
-    /* eslint-disable no-console */
-    console.time('Time taken to activate the extension');
-    console.log('Starting & waiting for Jupyter extension to activate');
-    await activateExtensionScript();
-    console.timeEnd('Time taken to activate the extension');
+    // for performance tests, extension activation is part of the test run
+    if (!IS_PERF_TEST) {
```

Comment:
Same thing here too. IS_PERF_TEST is a function:
https://github.com/microsoft/vscode-jupyter/blob/2b726d6a5b2ab045bcf32e4691baebc820b29fc6/src/test/constants.ts#L30